### ISSUE TITLE
Custom Properties Not Working

### ISSUE THREAD

**Issue by nullouis** [oasis-open/cti-python-stix2#102]
https://stix2.readthedocs.io/en/latest/guide/custom.html

I faced two types of error when trying to include custom properties:

1. TypeError: '<' not supported between instances of 'NoneType' and 'int'
2. stix2.exceptions.ExtraPropertiesError: Unexpected properties for Identity: (x_foo).

```
from stix2 import Identity, Bundle

identity2 = Identity(name="John Smith",
                     identity_class="individual",
                     x_foo="bar",
                     allow_custom=True)

print(identity2)		#1. Error encountered in Python 3
print(Bundle(identity2))	#2. Error encountered in Python 2, 3
```

**Comment by emmanvg** [comment]
Hi @nullouis! Thanks for the submission. If you cloned from GitHub, can you confirm that you have pulled the most recent changes of the repository?

If not, this issue was fixed in #85 and will be available in the next release of `stix2` (0.4) on PyPI soon. I hope this helps!

**Comment by nullouis** [comment]
The latest works perfectly. Thanks!
